abuild.in

```diff
@@ -837,6 +838,22 @@ check_maintainer() {
 	fi
 }
 
+check_license() {
+	local ret=0
+	local license_list=/usr/share/spdx/license.lst
+	if options_has "!spdx" || ! [ -f "$license_list" ]; then
+		return 0
+	fi
+	local spdx=$(cat "$license_list")
+	local i; for i in $license; do
+		if ! list_has "$i" $spdx; then
```

Comment:
I think I mentioned this before but imho using `grep -F -w "$i" "$license_list"` is much cleaner and could possibly be faster as well.